modules/n1ql/pages/n1ql-language-reference/transactions.adoc

```diff
@@ -184,194 +186,6 @@ When developing a transaction with an SDK, the transaction may contain a mixture
-----
-
-The results of running the transaction in the Query Workbench are shown below.
-If you are using the cbq shell, the results are formatted differently, but contain the same information.
-
-.Results
-[source,json]
-----
-include::example$transactions/results.jsonc[tags=!ellipsis]
-----
-
-<.> Beginning a transaction returns a transaction ID.
-<.> Before setting the second savepoint, the booking document has user `"0"`, name `"Keon Hoppe"`.
-<.> After setting the second savepoint and performing an update, the booking document has user `"1"`, name `"Rigoberto Bernier"`.
-<.> After rolling back to the second savepoint, the booking document again has user `"0"`, name `"Keon Hoppe"`.
-====
-
-[[ex-2]]
-.Check the results of <<ex-1>>
-====
-Check the result of committing the transaction.
-
-.Query
-[source,n1ql]
-----
-include::example$transactions/multiple.n1ql[tag=check-3]
-----
-
-.Results
-[source,json,indent=0]
-----
-include::example$transactions/results.jsonc[tag=check-3]
-----
-
-<.> The booking document has been added with the attributes that were present when the transaction was committed.
-====
-
-.Transaction using the Query REST API
-====
-For reference, this example shows the equivalent of <<ex-1>> using the Query REST API.
-
-.Begin transaction and set parameters
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=begin]
-----
-
-.Specify transaction settings
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=set]
-----
-
-.Create a booking document
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=upsert]
-----
-
-.Set a savepoint
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=savepoint-1]
-----
-
-.Update the booking document to include a user
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=update-1]
-----
-
-.Check the content of the booking and user
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=check-1]
-----
-
-.Set a second savepoint
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=savepoint-2]
-----
-
-.Update the booking documents to change the user
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=update-2]
-----
-
-.Check the content of the booking and user
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=check-2]
-----
-
-.Roll back the transaction to the second savepoint
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=rollback]
-----
-
-.Check the content of the booking and user again
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=check-3]
-----
-
-.Commit the transaction
-[source,console]
-----
-include::example$transactions/multiple.sh[tag=commit]
-----
-
-<.> After beginning the transaction, each subsequent statement in the transaction must specify the transaction ID that was generated when the transaction began.
-====
```

Comment:
Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)